epam_python_course/L4/CW/reduce_factorial.py

```diff
@@ -0,0 +1,8 @@
+"""
+посчитать факториал с помощью функции reduce
+
+"""
+import functools
+
+result = functools.reduce(lambda accum, n: accum * n if n != 0 else accum * 1, range(6), 1)
```

Comment:
why multiple accum on 1 in else branch? =)

Reply:
Agree! It is senselessly